contracts/src/v0.8/automation/dev/chains/OptimismModule.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
+
+import {OVM_GasPriceOracle} from "./../../../vendor/@eth-optimism/contracts/v0.8.9/contracts/L2/predeploys/OVM_GasPriceOracle.sol";
+import {IChainModule} from "../interfaces/v2_2/IChainModule.sol";
+
+contract OptimismModule is IChainModule {
+  /// @dev OP_L1_DATA_FEE_PADDING includes 35 bytes for L1 data padding for Optimism and BASE
+  bytes internal constant OP_L1_DATA_FEE_PADDING =
+    hex"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+  /// @dev OVM_GASPRICEORACLE_ADDR is the address of the OVM_GasPriceOracle precompile on Optimism.
+  /// @dev reference: https://community.optimism.io/docs/developers/build/transaction-fees/#estimating-the-l1-data-fee
+  address private constant OVM_GASPRICEORACLE_ADDR = 0x420000000000000000000000000000000000000F;
+  OVM_GasPriceOracle private constant OVM_GASPRICEORACLE = OVM_GasPriceOracle(OVM_GASPRICEORACLE_ADDR);
+
+  function blockHash(uint256 blocknumber) external view returns (bytes32) {
+    return blockhash(blocknumber);
+  }
+
+  function blockNumber() external view returns (uint256) {
+    return block.number;
+  }
```

Comment:
We can remove these after inheriting `ChainModuleBase()`

Reply:
done